Formula/mrbayes.rb

```diff
@@ -25,6 +25,7 @@ def install
     system "./configure", *args
     system "make"
     system "make", "install"
+    pkgshare.install [share/"documentation", share/"examples"]
```

Comment:
Does this work as desired? I didn't actually test this code.

Reply:
No. Locally I have changed `documentation` to `doc`, and _that_ works. All of the `configure` script was rewritten since the last stable release and we've opted for slightly different names of some of the directories. It's too late in the evening for me to work on this right now though... tomorrow.

Reply:
I'd suggest changing the location of the `examplesdir` to be under `pkgdatadir`.
```diff
-examplesdir=	${datarootdir}/examples/${PACKAGE}
+examplesdir=$(pkgdatadir)/examples
```
https://github.com/NBISweden/MrBayes/blob/develop/examples/Makefile.am#L1
See https://www.gnu.org/software/automake/manual/html_node/Standard-Directory-Variables.html
and https://www.gnu.org/software/automake/manual/html_node/Uniform.html

Reply:
It looks like the documentation is already in the right place.

Try this (untested):
```suggestion
    mv share/"examples/mrabyes", pkgshare/"examples"
```

Reply:
We may change the layout of the installation directories.  The current layout is based on how OpenBSD (my native Unix) is usually installing examples and PDF documentation.  Any change will happen in a future release though, so we'll go with moving the directory for the test for now. Thanks.